OpenRA.Mods.Common/Activities/Move/Move.cs

```diff
@@ -388,6 +390,10 @@ abstract class MovePart : Activity
 
 				IsInterruptible = false; // See comments in Move.Cancel()
 
+				TurnsWhileMoving = move.mobile.Info.TurnsWhileMoving;
+				if (TurnsWhileMoving)
+					FromToYaw = (To - From).Yaw;
```

Comment:
```suggestion
					FromToYaw = (To - From).Yaw + ActorFacingModifier;
```

This can be moved here as well. The modifier doesn't change while MovePart is running either way

Reply:
moved.